src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let cssProperties = sourceString.split(';');
+
+  cssProperties = cssProperties.map(prop => prop.split(':'));
+  cssProperties = cssProperties.map(prop => prop.map(x => x.trim()));
+  cssProperties = cssProperties.filter(prop => prop.length > 1);
+
+  const callback = (prev, x) => {
+    return {
+      ...prev,
+      [x[0]]: x[1],
+    };
+  };
```

Comment:
```suggestion
  const callback = (prev, x) => (
      ...prev,
      [x[0]]: x[1],
    };
  );
```
Відкриваєш круглі скобки, і js думає що в тебе функція в 1 строку. Тоді можна не писати return всередині